entities/entities/rvr_crafter/init.lua

```diff
@@ -23,6 +23,10 @@ function ENT:Initialize()
     RVR.Crafting.makeCrafter( self, self.CrafterName, self.Tier, self.CrafterType )
 end
 
+local nextUse = 0
```

Comment:
Why dont we just assign this to a variable on the entity

Reply:
its not used anywhere except in this single function